docs/guides/depositor-bot.md

```diff
@@ -0,0 +1,74 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of Deposit Security Module.
+
+The Depositor Bot obtains signed deposit messages from Council Daemons. Once a sufficient number of messages is
+collected to constitute a quorum, the bot proceeds to initiate a deposit into the designated staking module. This
+deposit is executed using the depositBufferedEther function within the "DepositSecurityModule" smart contract.
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
+
+- Ethereum EL node
+- Onchain transport node
```

Comment:
```suggestion
- Onchain databus transport node
```

Reply:
Add comment that currently it's gnosis chain

Reply:
Done